<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>371</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditVectors"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCalculate">
       <property name="text">
        <string>Calculate!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDotProduct">
       <property name="text">
        <string>Dot product between v1 and v2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSmallerAngle">
       <property name="text">
        <string>Smaller angle between v1 and v2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelOrthogonalVector">
       <property name="text">
        <string>Orthogonal vector of v1 and v2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelAreaOfTriangle">
       <property name="text">
        <string>Area of triangle defined by v1 and v2:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>414</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
